release-scripts/release.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash 
+
+set -e
+
+export BUILD_TAG=latest
```

Comment:
No need for this BUILD_TAG, just use the RELEASE_VERSION where is is used below.

Reply:
so a local/111:0.1.0 and nhsdev/nia-111-adaptor:0.1.0?